public/components/CapiImport/CapiImport.js

```diff
@@ -75,14 +75,21 @@ class CapiImport extends Component {
 
         this.setState({importing: false});
         this.context.router.push('/campaign/' + campaign.id);
-      })
+      }).catch((error) => {
+        console.error(error);
+        if (error.response) {
+          this.setState({importing: false, error: error.response});
```

Comment:
Is error.response here a message that is user friendly?

Reply:
Probably not. Shall I remove it? The message I saw was; `campaign does not exist`. But if it was something else it could be actually something not good.

Reply:
Maybe just give the user a message that they should try again as an issue occurred.